CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ The format is loosely based on [Keep a Changelog](http://keepachangelog.com/en/1
 
 #### Fixed
 
+* mock.Protected().SetupSet fails when argument is of type Expression (@tonyhallett, #1189)
```

Comment:
No it has always been an issue.  You changed the issue title from "ProtectedMock fails when arguments are of type Expression" to mock.Protected().SetupSet fails when argument is of type Expression.
I have just put the issue title in to the changelog.

Reply:
@tonyhallett, yes, the issue may always been present, but it never surfaced _for Moq users_ because `value` was ignored up until recently. So users don't need to know about this issue. The PR title is only relevant because it ends up in the commit history. The commit history is important to Moq developers, the changelog OTOH is important to Moq users. Please remove the changelog entry.

Reply:
The issue could have surfaced at any time over the last 12? years as ToExpressionArg was being called by ToExpressionArgs, called by the GetMethodCall methods ( `args` were not ignored ).  GetMethodCall methods are called by numerous paths.

For instance
https://github.com/moq/moq4/blob/a6fde8b6d79a7437bf642d115785b97f40779b6a/src/Moq/Protected/ProtectedMock.cs#L47

Link is from before #1186

Reply:
@tonyhallet, I see. Thank you for the explanation. Feel free to give this PR a more accurate / correct title.